examples/deepspeed-multinode/sky.yaml

```diff
@@ -0,0 +1,80 @@
+# Example: a distributed DeepSpeed job (DeepSpeed-Chat) on 2 VMs.
+#
+# This takes care constructing a "hostfile" to pass to DeepSpeed.
+#
+# Usage:
+#
+#   $ sky launch sky.yaml -r --down -c ds
+#
+#   # After the job starts running, log into the two nodes and check gpustat:
+#
+#   $ ssh ds
+#   $ gpustat -i
+#
+#   $ ssh ds-worker1
+#   $ gpustat -i
```

Comment:
Should we clarify that these steps are optional? Otherwise, it seems I have to run these commands.